.yamato/config/vv_perf_builtin.metafile

```diff
@@ -0,0 +1,255 @@
+      - [Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.kill_server>
+
+    commands_build:
+      - <reuse_blocks.unity_config>
+      - [iPhone_Metal]: pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
+      - [iPhone_Metal]: unity-downloader-cli <EDITOR_REVISION_CMD> -c editor -c iOS -c il2cpp   --wait --published-only
+      - [Win_DX11, Win_DX12, Win_Vulkan, Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.turn_off_firewall>
+      - [all]: <shared.platform.commands.setup_utr>
+      - [Android_Vulkan, Android_OpenGLES3, Win_DX11, Win_DX12, Win_Vulkan, OSX_Metal, Linux_Vulkan]: <shared.platform.commands.unity_downloader_cli>
+      - [iPhone_Metal, Android_Vulkan, Android_OpenGLES3, Win_DX11, Win_DX12, Win_Vulkan, OSX_Metal, Linux_Vulkan]: <shared.platform.commands.call_utr>
+        flags:
+          - <reuse_blocks.perf_utr_flags>
+
+  - type: playmode
+    name: performance_playmode_editor
+    commands:
+      - <reuse_blocks.unity_config>
+      - [Win_DX11, Win_DX12, Win_Vulkan, Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.turn_off_firewall>
+      - [all]: <shared.platform.commands.setup_utr>
+      - [all]: <shared.platform.commands.unity_downloader_cli>
+      - [all]: <shared.platform.commands.call_utr>
+        flags:
+          - [all]: --report-performance-data
+          - [all]: --performance-project-id=VikingVillage_BuiltIn
+          - [all]: --testfilter="MemoryAllocation;RuntimeProfilerMetrics"
+          - <reuse_blocks.perf_utr_flags>
+  - type: editmode
+    name: performance_build_editmode
+    commands:
+      - <reuse_blocks.unity_config>
+      - [Win_DX11, Win_DX12, Win_Vulkan, Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.turn_off_firewall>
+      - [all]: <shared.platform.commands.setup_utr>
+      - [all]: <shared.platform.commands.unity_downloader_cli>
+      - [all]: <shared.platform.commands.call_utr>
+        flags:
+          - [all]: --testfilter=Build
+          - [all]: --report-performance-data
+          - [all]: --performance-project-id=VikingVillage_BuiltIn
+          - <reuse_blocks.perf_utr_flags>
+expression_trigger:
+  expression: ""
+platforms:
+  - name: Win
+    apis:
+      - name: DX11
+      - name: DX12
+      - name: Vulkan
+        exclude_test_platforms:
+          - name: performance_playmode_editor
+    build_configs:
+      - name: mono
+    color_spaces:
+      - Linear
+    agents_project:
+      default:
+        type: Unity::VM::GPU
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+        model: rtx2080
+      standalone_build:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.xlarge
+      performance_build_editmode:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+        model: rtx2080
```

Comment:
I think the model is a typo? The agent is not using a GPU

Reply:
oh, thank you!